gpu/utils/include/pcl/gpu/utils/device/static_check.hpp

```diff
@@ -48,8 +48,10 @@ namespace pcl
     namespace device
     {
         template<bool expr> struct Static {};
-        
-        template<> struct Static<true> 
+
+        template <>
+        struct [[deprecated("This class will be replaced at PCL release 1.15  by "
```

Comment:
Please also add a deprecation for the file itself. `PCL_DEPRECATED_HEADER`

Reply:
Thanks for the comment, Kunal! I have added the deprecation notice. Is that what you had in mind?